schema/encoding/jsonschema/time.go

```diff
@@ -1,13 +1,12 @@
 package jsonschema
 
-import (
-	"io"
+import "github.com/rs/rest-layer/schema"
 
-	"github.com/rs/rest-layer/schema"
-)
+type timeBuilder schema.Time
 
-func encodeTime(w io.Writer, v *schema.Time) error {
-	ew := errWriter{w: w}
-	ew.writeString(`"type": "string", "format": "date-time"`)
-	return ew.err
+func (v timeBuilder) JSONSchema() (map[string]interface{}, error) {
+	m := make(map[string]interface{})
```

Comment:
avoid the allocation, package private var and then return it as is.
```
var timeJSONSchema map[string]interface{}
func init() {
    timeJSONSchema = make(map[string]interface{})
    timeJSONSchema["type"] = "string"
    timeJSONSchema["format"] = "data-time"
)
```

Then simply ```return timeJSONSchema, nil```